packages/medusa/src/api/routes/admin/discounts/get-condition.ts

```diff
@@ -0,0 +1,90 @@
+import { IsOptional, IsString } from "class-validator"
+import { MedusaError } from "medusa-core-utils"
+import {
+  defaultAdminDiscountConditionFields,
+  defaultAdminDiscountConditionRelations,
+} from "."
+import { DiscountCondition } from "../../../../models"
+import { DiscountService } from "../../../../services"
+import DiscountConditionService from "../../../../services/discount-condition"
+import { getRetrieveConfig } from "../../../../utils/get-query-config"
+import { validator } from "../../../../utils/validator"
+/**
+ * @oas [get] /discounts/{discount_id}/conditions/{condition_id}
+ * operationId: "GetDiscountsDiscountConditionsCondition"
+ * summary: "Gets a DiscountCondition"
+ * x-authenticated: true
+ * parameters:
+ *   - (path) discount_id=* {string} The id of the Discount.
+ *   - (path) condition_id=* {string} The id of the DiscountCondition.
+ * query:
+ *  - (query) expand {string} Comma separated list of relations to include in the results.
+ *  - (query) fields {string} Comma separated list of fields to include in the results.
+ * description: "Gets a DiscountCondition"
+ 
+ * tags:
+ *   - DiscountCondition
+ * responses:
+ *   200:
+ *     description: OK
+ *     content:
+ *       application/json:
+ *         schema:
+ *           properties:
+ *             discount_condition:
+ *               $ref: "#/components/schemas/discount_condition"
+ */
+
+export default async (req, res) => {
+  const { discount_id, condition_id } = req.params
+
+  const validatedParams = await validator(
+    AdminGetDiscountsDiscountConditionsConditionParams,
+    req.query
+  )
+
+  const discountService: DiscountService = req.scope.resolve("discountService")
+
+  const discount = await discountService.retrieve(discount_id, {
```

Comment:
Don’t think we need to select everything from the rule and rule.conditions wdyt just to select there ids since it is only to check the existence

Reply:
This is a good point although I would consider this non-blocking